scripts/wait_polkadot_launch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+while true; do
+    sleep 1
+    stdout=$(docker-compose logs --tail=1 | grep "POLKADOT LAUNCH COMPLETE")
```

Comment:
are there not logs after that message? just not to miss one. can we show some value while waiting? or will see hang?

Reply:
this is the last message in logs. other messages are about configuration parameters of relay chain and collators, i.e. duplicate the [config](https://github.com/ComposableFi/composable/blob/andor0/add-runtime-tests-in-ci/scripts/polkadot-launch/composable_and_basilisk.json).

Reply:
if message will never appear, what i will see?

Reply:
Probably worth printing the logs until we get the LAUNCH complete?

Reply:
> if message will never appear, what i will see?

a message about timeout

Reply:
> Probably worth printing the logs until we get the LAUNCH complete?

done
[example](https://github.com/ComposableFi/composable/runs/6055268569?check_suite_focus=true)